crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -620,6 +620,79 @@ mod private {
             Ok(Self { room: room_id, store, events, waited_for_initial_prev_token: false })
         }
 
+        /// Load more events backwards if the last chunk is **not** a gap.
+        ///
+        /// Return `Ok(Some((events, reached_start), _))` if events have been
+        /// inserted, `Ok(None)` if a gap has been inserted or if the
+        /// store is disabled.
```

Comment:
`Ok(None)` could also mean that there's no previous events. Hopefully the callers can handle that properly too :sweat_smile:

Reply:
Correct. I need to update the comment!